ansible/roles/master/tasks/main.yml

```diff
@@ -70,6 +70,14 @@
 - name: write the kubecfg (auth) file for kubectl
   template: src=kubectl.kubeconfig.j2 dest={{ kube_config_dir }}/kubectl.kubeconfig
 
+- name: write the config files for kubelet
+  template: src=kubelet.j2 dest={{ kube_config_dir }}/kubelet
```

Comment:
They are different. The master kubelet must not register itself and runs different options.